README.md

```diff
@@ -1 +1,92 @@
-# JSCapstone
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Js Capstone
+
+> In this project I'm practicing how to use RESTful API's. I'm using GET and POST methods to either display shows on the screen or to Display data that's related to the Involvement API that Microverse built for this learning project.
+
+Basic UI of the project:
+
+![screenshot](./screenshot1.png)
+
+UI of the Pop Up:
+![screenshot](screenshot2.png)
+
+## Built With
+
+- Js
+- Html
+- Css
+
+## Live Demo
+
+[Live Demo Link](https://nelsinofrancisco.github.io/JSCapstone/)
+
+## Project Description (Loom Video)
+
+Check out the [video](https://www.loom.com/share/cfc64e65a3af494998491ee142811a67)
```

Comment:
**Changes Required**
- [ ] Good presentation, I love your composure. However, you just missed one thing in the presentation.
**The video contains a highlight about a piece of the code the student finds interesting or is proud of making.**  _I am afraid you will have to record another video._

Reply:
Thank you for pointing it out, I did the change.